### PR TITLE
Skip validate_view_consistency for nvFuser tests

### PR DESCRIPTION
**Pull request by IvanYashchuk**
nvFuser's execute function always returns a copy for now.

Ref. https://github.com/pytorch/pytorch/pull/84629#discussion_r966375582

### CODE REVIEW

File: test/test_ops.py [hunk 1]
```diff
@@ -379,6 +379,7 @@ def test_python_ref_executor(self, device, dtype, op, executor):
             skip_zero_numel=("nvfuser" in executor),  # nvfuser doesn't support zero-sized tensors
             skip_zero_dim=skip_zero_dim,
             skip_bfloat=("nvfuser" in executor),  # nvfuser doesn't support bfloat tensors for pre-11 cuda TK
+            skip_view_consistency=("nvfuser" in executor),  # nvfuser doesn't support view consistency
```
Comment (mruberry) [inline test/test_ops.py:382]: Let's file an issue and comment its number here to track this

**Review by mruberry — APPROVED** [review]
Cool, but see inline comment and this will probably need to retrigger the lint job

**Comment by mruberry** [comment]
> ## Merge failed
> **Reason**: Approval needed from one of the following (Rule 'superuser'): zanqi, izaitsevfb, andysamfb, muchulee8, carolineechen, ...
> 
> Details for Dev Infra team

Ug, @malfet -- can we update this list?

**Review by ngimel — APPROVED** [review]

**Pull request closed without merging**